test/fgpu/test_engine.py

```diff
@@ -47,18 +48,23 @@
 SYNC_EPOCH = 1632561921
 CHANNELS = 1024
 SPECTRA_PER_HEAP = 256
-# Lower than the default to make tests quicker, and smaller than
-# CHUNK_JONES to check that this case works.
+# Lower than the default to make tests quicker
 # TODO: use a number that's not a multiple of the number of channels,
 # once _send_data can handle partial chunks.
 CHUNK_SAMPLES = 524288
-CHUNK_JONES = 1048576
+CHUNK_JONES = 131072
 MAX_DELAY_DIFF = 16384  # Needs to be lowered because CHUNK_SAMPLES is lowered
 PACKET_SAMPLES = 4096
 TAPS = 16
 FENG_ID = 42
 ADC_SAMPLE_RATE = 1712e6
 
+WIDEBAND_ARGS = f"name=test_wideband,dst=239.10.11.0+15:7149,taps={TAPS}"
+# Centre frequency is a multiple of the channel width
```

Comment:
Is that true regardless of the number of channels?

Reply:
Probably not - just for the specific CHANNELS constant defined here.